plugins/orchestrator-backend/src/service/api/v2.ts

```diff
@@ -75,19 +78,21 @@ export class V2 {
   }
 
   public async getInstances(
-    pagination: Pagination,
+    pagination?: Pagination,
+    filter?: FilterInfo,
   ): Promise<ProcessInstanceListResultDTO> {
     const instances = await this.orchestratorService.fetchInstances({
       pagination,
+      filter,
     });
     const totalCount =
       await this.orchestratorService.fetchInstancesTotalCount();
 
     const result: ProcessInstanceListResultDTO = {
       items: instances?.map(mapToProcessInstanceDTO),
       paginationInfo: {
-        pageSize: pagination.limit,
-        page: pagination.offset,
+        pageSize: pagination?.limit,
+        page: pagination?.offset,
```

Comment:
this should be pagination?.offset / pageSize

Reply:
ok, now i get it. Thanks for clarifying.
I'll change from `page` to `offset` to be, also,  "compliant" with graphql syntax.

Reply:
done